### DISCUSSION TITLE
Palworld Server Options

### DISCUSSION THREAD

**Discussion by firepro20** [GameServerManagers/LinuxGSM#4455] [Q&A]
I was thinking of hosting a Palworld server but I am not sure what the best option would be given that I've read a couple posts online that there are currently memory leaks related to world exploration. There is quite a difference in price from getting a Linux distro server or a Windows based server, and compatibility in terms of minimum specs for the game, let's say it will be for 4-6 people at once.

Any ideas or suggestions? I was looking at Contabo hosting VPS.

**Comment by fizikiukas** [comment]
I would go with Hostinger - https://www.hostinger.com/minecraft-server-hosting
It comes with a Game Panel, which will allow you to create a Palworld server through the browser. Or any other out of 170 game servers.
